docs/geedocs/answer/7160000.html

```diff
@@ -148,10 +149,95 @@
           <td>Close the resource form and open it again to make the save option available again.</td>
         </tr>
         <tr>
+          <td>340</td>
+          <td>GE Fusion Terrain is black</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
           <td>342</td>
           <td>Fusion crashes when opening an unsupported file type</td>
           <td>Re-open fusion and avoid opening unsupported file types.</td>
         </tr>
+        <tr>
+          <td>375</td>
+          <td>unbundle psycopg2</td>
+          <td>On Ubuntu 16.04 switch to python-psycopg2' instead of 'python2.7-psycopg2'.</td>
+        </tr>
+        <tr>
+          <td>380</td>
+          <td>Provider field in resource-view is blank</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>401</td>
+          <td>GEE commands are not in the path for sudo. </td>
+          <td>Add /opt/google/bin to the path for all users, including the super user</td>
+        </tr>
+        <tr>
+          <td>402</td>
+          <td>Provider manager window locked to main window.</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>403</td>
+          <td>Missing close button on system manager window in RHEL 7</td>
+          <td>Right click the title bar and select close</td>
+        </tr>
+        <tr>
+          <td>404</td>
+          <td>Opaque polygons in preview.</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>405</td>
+          <td>Vector layer preview not cleared in some situations</td>
+          <td>Reset the preview window to the correct state by either clicking on it or previewing another vector layer</td>
+        </tr>
+        <tr>
+          <td>407</td>
+          <td>Corrupt data warning when starting fusion</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>423</td>
+          <td>Improve JPEG 2000 format support performance</td>
```

Comment:
Minor: I'd reword this to something like "JPEG 2000 performance is slow" or something like that.

Reply:
Even if the original PR (423) title is 'Improve JPEG 2000 format support performance' ?

Reply:
I haven't necessarily been keeping the issue title and the wording in the release notes the same, though that's not a bad idea.  The point I want to get across is that jp2 performance will be worse in 5.2.0 than it was in 5.1.3.

Reply:
yeah, the title does not indeed reflect that. may be we leave it as is and if one interested to learn more they would look at the PR and understand the issue ? or we can change the title here ?

Reply:
I'd like to change the title here - maybe "Slower JPEG2000 performance than 5.1.3" and then if they want to know more they can look at the issue on github.

Reply:
ok